buildtools/composer/composer.go

```diff
@@ -68,7 +107,7 @@ func (c *Composer) Dependencies(dir string) ([]Package, map[Package][]Package, e
 		}
 
 		// We're somewhere in the tree.
-		r := regexp.MustCompile("^([ \\|`-]+)([^ \\|`-][^ ]+) (.*)$")
```

Comment:
I'm not familiar with this parsing method, is there a reason you removed the two backslashes?

Reply:
The escaping is redundant: https://godoc.org/regexp/syntax and https://github.com/google/re2/wiki/Syntax